src/php/Printer/Printer.php

```diff
@@ -126,6 +127,9 @@ private function createObjectTypePrinter($form): TypePrinterInterface
         if (method_exists($form, '__toString')) {
             return new ToStringPrinter();
         }
+        if (is_callable($form)) {
```

Comment:
I we only want to check for Phel function we should check if `$form` is an instance of `FnInterface`. Otherwise `php/substr` would also print `<function>`.

Don't know what's better.

Reply:
I think it's a good idea to use `if ($form instanceof FnInterface) {...`.